modules/serverless-list-source-types.adoc

```diff
@@ -0,0 +1,25 @@
+// Module included in the following assemblies:
+//
+// * serverless-kn-source.adoc
+
+[id="serverless-list-source-types_{context}"]
+= Listing available event source types using `kn`
+
+You can list the available event source types in the terminal by using the following command:
+----
+$ kn source list-types
+----
+
+The default output for this command will look like:
+----
+TYPE              NAME                                            DESCRIPTION
+ApiServerSource   apiserversources.sources.knative.dev            Watch and send Kubernetes API events to a sink
+PingSource        pingsources.sources.knative.dev                 Send periodically ping events to a sink
```

Comment:
PingSource description sounds odd:
s/Send periodically ping events/Periodically send ping events
-or-
s/Send periodically ping events/Send ping events periodically

Reply:
Sorry, fixed this one in the next commit